sync/src/relayer/compact_block_process.rs

```diff
@@ -259,25 +259,14 @@ impl<'a> CompactBlockProcess<'a> {
         } else {
             StatusCode::CompactBlockRequiresFreshTransactions.with_context(&block_hash)
         };
-        if !missing_transactions.is_empty() {
-            metrics!(value, "ckb-net.fresh", missing_transactions.len() as u64, "type" => "transactions", "status" => status.tag());
-        }
-        if !missing_uncles.is_empty() {
-            metrics!(value, "ckb-net.fresh", missing_uncles.len() as u64, "type" => "uncles", "status" => status.tag());
-        }
 
         let content = packed::GetBlockTransactions::new_builder()
             .block_hash(block_hash)
             .indexes(missing_transactions.pack())
             .uncle_indexes(missing_uncles.pack())
             .build();
         let message = packed::RelayMessage::new_builder().set(content).build();
-
-        if let Err(err) = self.nc.send_message_to(self.peer, message.as_bytes()) {
-            return StatusCode::Network
-                .with_context(format!("Send GetBlockTransactions error: {:?}", err));
-        }
-        crate::relayer::metrics_counter_send(message.to_enum().item_name());
+        let _ignore = send_message_to(self.nc.as_ref(), self.peer, &message);
```

Comment:
It's better to add a log on error, and use the log message to explain why this error can be ignored.

Reply:
updated.